R/req-error.R

```diff
@@ -33,5 +33,14 @@ error_is_error <- function(req, resp) {
 }
 
 error_body <- function(req, resp) {
-  req_policy_call(req, "error_body", list(resp), default = NULL)
+  tryCatch(
+    req_policy_call(req, "error_body", list(resp), default = NULL),
+    error = function(cnd) {
+      cnd <- cnd_entrace(cnd)
```

Comment:
`cnd_entrace()` will not help much in a `tryCatch()` handler, but also you seem to return a character vector so the entracing is ignored?